packages/manager/modules/pci/src/projects/project/private-registry/onboarding/index.js

```diff
@@ -0,0 +1,66 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import component from './onboarding.component';
+
+const moduleName = 'ovhManagerPciPrivateRegistryOnboardingLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+  ])
+  .config(/* @ngInject */ ($stateProvider) => {
+    $stateProvider
+      .state('pci.projects.project.private-registry.onboarding', {
+        url: '/onboarding',
+        component: 'pciProjectPrivateRegistryOnboarding',
+        params: {
+          registryId: null,
+        },
+        resolve: {
+          breadcrumb: () => null, // Hide breadcrumb
+        },
+        translations: {
```

Comment:
You should remove `translations` from states and use `@ngTranslationsInject`. Translations are injected/loaded and we have a `$translate.refresh()` on `pci.*` transitions (cf: https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/index.js#L89)